ReadAlongs/Studio-Web
# feat: connect studio tour to editor tour (#346)

* feat: connect studio tour to editor tour

* chore: updated translations

* fix: implemented feedback from Eric

* chore: translations for the tour connection to the editor

---------

Co-authored-by: Eric Joanis <eric.joanis@nrc-cnrc.gc.ca>
(cherry picked from commit 8b89a0381cc52bc8fa0fd97ef735b145b3b73750)



### PR Goal? 

Redo #346 on top of the reverted main branch.

### Fixes? 

Connects the editor tour at the end of the studio tour

### Feedback sought? 

testing, checking the code, making sure it doesn't break the editor

### Priority? 

medium

### Tests added? 

no

### How to test? 

Check the PR previous in the three languages, click to take the tour from Studio, and see that you get the option to continue on to the Editor tour at the end.

### Confidence? 

high

### Version change? 

no
